### PR TITLE
Update: Surveys and Observations

### LINKED ISSUE #188: Rationalise and Consolidate: Survey type, Survey method, Observation type, Observation instrument
We need to fix the vocabs associated with geophysical surveys following the resolution of multi-type surveys into a single survey (with the raw aquired data attached) to cover the acquisition and then multiple observations of different type (with the processed datasets attached). The processsed datasets can also be related to the original survey as additional datasets where their survey 'type' is recorded by their earth science data category.  

E.g. AG surveys are therefore of type:geophysics and method:airborne. There would also be mag, rads, and DEM observations with the grids/images attached. The Mag/RAD/DEM observation metadata could be omitted if it is not of interest and all datasets could be attached to the original AG survey.

The changes required involves substantial expansion of observation instrument to include AG tech specs and seismic source along with the additional concepts from the survey type review. Additional work will be required to transfer detail from survey type to observation type, remove narrower concepts from survey method, understand how/if obs.sample_preparation will be used in Geophysical surveys, workshop how the new understanding of AG surveys is reflected in Seismic, geochem, general field sampling, etc., possibly writing some documentation along the way. 

@CantRoger might be the right person to lead this. I'll raise this as an activity in the standup to prioritise this work.

### LINKED ISSUE #206: Survey type needs concepts removed
Following the discussion about survey-observation-collection, concepts need to be removed and added to observation_type

### PR DESCRIPTION
**Pull request by KellyVance**
Updates to:
- Survey Type
- Survey Method
- Geological Observation Type
- Geological Observation Method

closes #206 
closes #188 

As always these are intended to be incremental change.  Further additions and augmentations to be added via future pull requests, during the development and UAT phase.

Please review as per roles below (you may have multiple roles). 
This will count as approved with any approval combination that provides 2 SMEs, BOTH owners, and 1 technical review.

Reviewers
SMEs : @CantRoger @Greenwoodmatthew @LukeHauck
Vocab Owners: @Greenwoodmatthew (Surveys), @geoderekh (Observations)
Technical Review: @LukeHauck, @nicholascar, or @DavidCrosswellGSQ 

FYI - Vocabulary Changes @lisa-woody

### CODE REVIEW

**Comment by LukeHauck** [comment]
Tech is good.

File: vocabularies/geological-observation-method.ttl [hunk 1]
```diff
@@ -1,43 +1,461 @@
...
+obsmh:mt-methods a skos:Collection ;
+    skos:definition "The classification of methods used in magnetotelluric geophysical survey acquisition."@en ;
+    skos:member obsmh:lpmt,
+        obsmh:csem,
+        obsmh:amt,
+        obsmh:bbmt ;
+    skos:prefLabel "Magnetotelluric Methods"@en .
+
+obsmh:spectral-methods a skos:Collection ;
+    skos:definition "The classification of methods used in magnetotelluric geophysical survey acquisition."@en ;
```
Comment (CantRoger) [inline vocabularies/geological-observation-method.ttl:449]: Change to "The classification of methods used in **spectral** survey acquisition."

Comment (KellyVance) [reply vocabularies/geological-observation-method.ttl]: Good pick-up. Thanks! Fixed.

File: vocabularies/geological-observation-method.ttl [hunk 2]
```diff
@@ -1,43 +1,461 @@
...
+    
+obsmh:mt-methods a skos:Collection ;
+    skos:definition "The classification of methods used in magnetotelluric geophysical survey acquisition."@en ;
+    skos:member obsmh:lpmt,
+        obsmh:csem,
+        obsmh:amt,
+        obsmh:bbmt ;
+    skos:prefLabel "Magnetotelluric Methods"@en .
+
+obsmh:spectral-methods a skos:Collection ;
```
Comment (geoderekh) [inline vocabularies/geological-observation-method.ttl:448]: Here "Spectral" is used in place of "Remote Sensing", so I suggest renaming collection to Remote Sensing.  LIDAR RADAR etc are remote sensing methods but are not spectral methods. I question the need for a generic interferometer here also. Also, remove XRF from collection. The fact that these all utilise EMR as a stimulus or proxy for measuring various properties shouldn't be the basis for this collection.

File: vocabularies/survey-type.ttl [hunk 3]
```diff
@@ -172,128 +83,23 @@ svytp:physical-geology a skos:Concept ;
...
-    skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
-    skos:notation "gv" ;
-    skos:prefLabel "Gravity"@en ;
-    skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
-
-svytp:magnetic a skos:Concept ;
-    skos:altLabel "aeromagnetics"@en ;
-    skos:definition "A survey where the variations of the Earth's magnetic field is measured to determine the distribution of magnetic minerals (predominately magnetite)"@en ;
+    
+svytp:spectral a skos:Concept ;
```
Comment (geoderekh) [inline vocabularies/survey-type.ttl:101]: The remote sensing community would be puzzled by this categorisation. Also, in observation there are a bunch of things that don't fit this definition and, arguably, aren't "spectral' surveys. All are resolved if we simply call these remote sensing surveys?

File: vocabularies/survey-type.ttl [hunk 4]
```diff
@@ -172,128 +83,23 @@ svytp:physical-geology a skos:Concept ;
...
+    skos:definition "A survey capturing image information and intensity-per-wavelength-band to be used for spectral analysis e.g. ASTER, MODIS."@en ;
     skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
-    skos:notation "mg" ;
-    skos:prefLabel "Magnetic"@en ;
-    skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
+    skos:prefLabel "Spectral"@en ;
```
Comment (geoderekh) [inline vocabularies/survey-type.ttl:104]: Doesn't this need a notation?

File: vocabularies/survey-type.ttl (hunk 3 above)
Comment (Greenwoodmatthew) [reply vocabularies/survey-type.ttl:101]: lidar and radar would be Location Survey i think in this classification.  Although i would be happy to resolve these both into Remote sensing if others think it makes more sense?

File: vocabularies/geological-observation-method.ttl (hunk 2 above)
Comment (Greenwoodmatthew) [reply vocabularies/geological-observation-method.ttl:448]: as in survey type above i would be ok with remote sensing

Comment (KellyVance) [reply vocabularies/geological-observation-method.ttl:448]: Spectral links through to the spectral survey type. As per the other collections.
More accurately there should be a spatial collection and concepts should be moved into it, including synthetic aperture radar, radar, lidar, and alser altimetry.

File: vocabularies/survey-type.ttl (hunk 3 above)
Comment (KellyVance) [reply vocabularies/survey-type.ttl:101]: As per above comment. some concepts need to be moved to a spatial (location survey) collection. This creates a logical link through to survey type in line with the other collections.

The inclusion of radar and lidar in that collection was a mistake that has lead people to a false idea that spectral was intended to be synonymous with remote sensing. Spectral and Spatial are the data types here, the survey method satellite, airborne etc would inform you on if that data was remotely sensed

File: vocabularies/survey-type.ttl (hunk 4 above)
Comment (KellyVance) [reply vocabularies/survey-type.ttl:105]: Yep! Added.

File: vocabularies/survey-type.ttl (hunk 3 above)
Comment (Greenwoodmatthew) [reply vocabularies/survey-type.ttl:101]: Thanks Vance, i think i am more comfortable with having the spectral / location survey as current.  Means we can separate geophysical activities and imagining (spectral) activities and spatial (lidar etc).  

Bu having spectral as a type means we can link between hylog and hymap surveys as discussed with Roger
Survey Type:                      Spectral                                             Spectral
Survey Method:                Airborne                                            Laboratory
Observation Type:           Spectral                                             Spectral                                             
Observation Method:     Multispectral                                    Hyperspectral
Instrument:                        ASTER                                                 Hylogger

**Comment by KellyVance** [comment]
> Would like a couple of changes in relation to remote sensing surveys/observations, and one missed notation in surv_type in an otherwise excellent commit.

notation added

See discussion re collections: misattribution of concepts and omission of one collection

**Pull request merged**